draft-poirier-rats-eat-da.md

```diff
@@ -57,35 +59,82 @@ From a trust perspective, DA allows a device to be included in the TVM's Trusted
+Each individual device claim is the combination of a device name and a standard claims format based on the bus or protocol the device supports.
+As previously mentioned, this draft currently defines the claims set for SPDM compliant devices and PCIe legacy devices that do not support the SPDM protocol.
+Careful condideration was also given to the overall design in order to leave room for future expansion.
+
 ~~~ cddl
 {::include-fold cddl/da-token.cddl}
 ~~~
 
 ## SPDM Claims
 
+A SPDM claim instance is expected to be present for each SPDM compatible device to be attested.
+Each instance consists of measurements and a certificates section.
+There can be up to 239 measurements per device with the entire measurement log optionally signed by the certificate populated in one of the 8 certificate slots.
+It should be noted that measurements formalized herein follow the DMTF measurement specification.
+
 ~~~ cddl
 {::include-fold cddl/spdm-claims.cddl}
 ~~~
 
 ### Measurement Claims
 
+SPDM measurements start with a component type that reflects one of the 10 categories defined by the SPDM specification.
+Following is the measurement itself represented by either a raw bitstream or a digest.
+The size of the digest value is derived from the measurement hash algorithm conveyed by the SPDM ALGORITHMS message response.
+
 ~~~ cddl
 {::include-fold cddl/spdm-measurement.cddl}
 ~~~
 
+### Measurement Claims Signature
+
+SPDM compliant devices can optionally support the capability to sign measurements.
+Included in the measurement claim signature are all the elements needed by a third party entity to reconstruct the original measurement log signed by the device.
+Those elements include L1 (see CDDL below), the combined SPDM prefix, the hash algorithm used to generate a digest of the measurement log and nonces provided by the requester and responder.
+The slot number of the leaf certificate used to sign the measurement log is also provided.
+
+~~~ cddl
+{::include-fold cddl/spdm-measurement-blocks-signature.cddl}
+~~~
+
 ### Certificate Claims
 
+According to the specification, SPDM compliant devices should support at most 8 slots, with slot 0 populated by default.
+Slot 0 SHALL contain a certificate chain that follows the Device certificate model or the Alias certificate model.
+Regardless of the certificate model used, a certificate chain comprises one or more DER-encoded X.509 v3 certificates {{-pkix}}.
+The certificates MUST be concatenated with no intermediate padding.
+
 ~~~ cddl
 {::include-fold cddl/spdm-certificates.cddl}
 ~~~
 
+## PCIe Legacy Device Claims
+
+The definition of a device claims format for PCIe legacy devices that do not implement the extensions needed to attest for their provenance and configuration is provided, making it is possible to keep using current assets as secures ones are being provisioned.
```

Comment:
In the introduction, we call this a “claims set”, following EAT conventions.  I reckon we should stick to that and `s/claims format/claims set/g`